ios/gamma/stack/RNSStackScreenComponentView.mm

```diff
@@ -18,6 +23,9 @@ @interface RNSStackScreenComponentView () <RCTMountingTransactionObserving>
 @implementation RNSStackScreenComponentView {
   RNSStackScreenController *_Nonnull _controller;
   RNSStackScreenComponentEventEmitter *_Nonnull _reactEventEmitter;
+#ifdef RCT_NEW_ARCH_ENABLED
+  RNSInvalidatedComponentsRegistry *_Nonnull _invalidatedComponentsRegistry;
```

Comment:
Why do we keep "whole registry" here, when only this one single instance can be pushed there?